setup.cfg

```diff
@@ -35,6 +35,8 @@ markers =
     tes
     torque
     wes_server
+    cwl_small_log_dir
```

Comment:
Does this mark mean anything *about* the tests (like they need something) or is it just a category of tests so we can run them?